feature(Jwt): Make cookie from jwtFilter conditional

gluizbr opened the pull request.

symphony-thibault reviewed (commented):
Hi @gluizbr, could you maybe document this in https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/tree/master/symphony-bdk-legacy/symphony-bdk-bot-sdk-java#configuration? 

As well, would it be possible to add a unit test that check that the `JwtCookieFilter` is not injected in the Spring context when prop `jwt-cookie.enable` is set to `true`?

gluizbr commented:
> Hi @gluizbr, could you maybe document this in https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/tree/master/symphony-bdk-legacy/symphony-bdk-bot-sdk-java#configuration?
> 
> As well, would it be possible to add a unit test that check that the `JwtCookieFilter` is not injected in the Spring context when prop `jwt-cookie.enable` is set to `true`?

I've updated the documentation and added some UT to check if the JwtCookieFilter is disabled when the property is false

marcus-secato-symphony commented on symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/main/java/com/symphony/bdk/bot/sdk/webapi/security/config/SecurityConfig.java:
```diff
@@ -1,94 +1,95 @@
+  }
+
+  @Bean
+  @ConditionalOnProperty(prefix = "jwt-cookie", name = "enable", havingValue = "true", matchIfMissing = true)
```
matchIfMissing = false as we do not want it enabled by default

gluizbr replied:
done

gluizbr commented on symphony-bdk-legacy/symphony-bdk-bot-sdk-java/README.md:
````diff
@@ -120,6 +120,8 @@ Symphony Bot SDK requires two main configuration files, ```application.yaml``` (
 
 It is required to set the path to ```bot-config.json``` in ```application.yaml``` through the ```bot-config``` property, as well as, you must set the path to the RSA private key using ```certs``` property.
 
+It's posible to disable JwtCookie on authentication endpoint adding the jwt-cookie enable property
````
done

symphony-thibault reviewed (approved):
LGTM 👍
